src/features/pages/markets/etf/carousel-markets/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import { Autoplay } from 'swiper'
+import * as styles from './styles.module.scss'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+import CardBasic from 'features/components/atoms/card/card-primary'
+import { CardType } from 'features/components/atoms/card/card-primary/type'
+
+interface MarketsMainSliderProps {
+    cards: CardType[]
+}
+
+const MarketsMainSlider = ({ cards }: MarketsMainSliderProps) => {
+    const { is_mobile_or_tablet } = useBreakpoints()
```

Comment:
In this case, [visibility.scss](https://github.com/binary-com/deriv-com/blob/master/src/features/styles/theme/visibility.scss) would be better here.

Reply:
we are using conditions here.so used breakpoints